pytorch_lightning/trainer/training_io.py

```diff
@@ -274,12 +273,7 @@ def _atomic_save(self, checkpoint, filepath: str):
         # Can't use the new zipfile serialization for 1.6.0 because there's a bug in
         # torch.hub.load_state_dict_from_url() that prevents it from loading the new files.
         # More details can be found here: https://github.com/pytorch/pytorch/issues/42239
-        if LooseVersion(torch.__version__).version[:3] == [1, 6, 0]:
-            torch.save(checkpoint, bytesbuffer, _use_new_zipfile_serialization=False)
-        else:
-            torch.save(checkpoint, bytesbuffer)
-        with cloud_open(filepath, 'wb') as f:
-            f.write(bytesbuffer.getvalue())
+        atomic_save(checkpoint, filepath)
```

Comment:
can you remove the whole method _atomic_save? It seems pointless to have it. You anyway copied the docstring and the bytesbuffer variable above is unused.